### ISSUE TITLE
a-sky broken in 0.8.0

### ISSUE THREAD

**Issue by i3games** [aframevr/aframe#3428]
**Description:**

- A-Frame Version:
0.8.0

- Platform / Device:

Chrome and Firefox / mac

- Reproducible Code Snippet or URL:

```
<a-scene>
       <a-sky color="#030303"></a-sky>
</a-scene>
```

Sky is not rendered, white page instead. Works in 0.7.1

Console Warrnigs:
```
THREE.WebGLRenderer: .shadowMap.renderReverseSided has been removed. Set Material.shadowSide instead.
aframe-v0.8.0.js:50683:5
THREE.WebGLRenderer: .shadowMap.renderSingleSided has been removed. Set Material.shadowSide instead.
```

**Comment by stephanedemotte** [comment]
quick fix `sky.object3DMap.mesh.material.side = THREE.BackSide`

**Comment by SteveX** [comment]
Same for my scenes as well.
Where would I apply that quick fix? Or is this something I should wait for in the next version?

**Comment by dmarcos** [comment]
Two work arounds:

1. If you are applying a color to the background use the background component on `a-scene`: https://aframe.io/docs/0.8.0/components/background.html#sidebar

2. You can build your own entity instead of using the `a-sky` primitive:

```
<a-entity material="src: #yourImage; side: back; shader: flat" 
                geometry="primitive: sphere; radius: 5000; segmentsWidth: 64; segmentsHeight: 32">
</a-entity>
```

**Comment by i3games** [comment]
Why is this issue closed? Do you intend to fix the bug?

**Comment by dmarcos** [comment]
@i3games The bug is fixed now on master and it will ship in the next A-Frame version. You can use the workarounds [described above](https://github.com/aframevr/aframe/issues/3428#issuecomment-372076295) or use the master branch in your application (https://rawgit.com/aframevr/aframe/master/dist/aframe-master.min.js)

**Comment by i3games** [comment]
Ok, cool.

**Comment by SteveX** [comment]
Thanks!

**Comment by ngokevin** [comment]
I've hot-patched the fix into 0.8.0. So check it now!

**Comment by i3games** [comment]
`<a-sky>` is back :) Thanks.
